requirments/REQUIREMENTS.md

```diff
@@ -0,0 +1,35 @@
+## Roles
+#### Unauthorized user
+Unathorized users can view books and comments to them, but cannot purchase books or subscriptions. Also they can sign in.
+#### Authorized user
+Authorized users can view books, purchase books and subscriptions, view and leave comments. Also they can sign out.
+#### Moderator
+Moderators can view, edit, leave and remove comments to books.
+#### Copyright holder
+Copyright holders can agree with a service manager to add or remove a book to/from the service.
+#### Service manager
+Service managers have contacts of copyright holders and can agree with them to add/remove books.
+
+## Use case
+![use cases](https://github.com/kkarnauk/se-project-fall-2021/blob/task_6/img/useCase.jpg)
```

Comment:
From the Use Case diagram, it looks like both moderators and users are able to remove any comments...
Also, I am not quite sure that `Pay Cart` is actually `extending` `View Cart`…
What happens when a user purchases a subscription? How does it affect all other parts of pipeline?